beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md

```diff
@@ -0,0 +1,286 @@
+# XDM implementation
+
+{% hint style="warning" %}
+The Adobe Experience Platform Edge mobile extension is currently in BETA. Use of this extension is by invitation only. Please contact your Adobe Customer Success Manager to learn more and get access to the materials for this tutorial.
+{% endhint %}
+
+## Prerequisites for this tutorial
+
+* Access to Adobe Experience Platform
+* Access to AEP Real-time Customer Profile
+* Minimal Swift / Android development knowledge 
+* Knowledge about the AEP Edge extension
+* Completion of [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup).
+
+### Download the sample application
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+Clone or download the Android Sample application from [https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-3](https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-3).
+
+To get started, follow the steps described in [AEP SDK Sample App Android - Installation](https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-1#installation).
```

Comment:
There is no `beta-assignment-1` branch for this repo, but there is a tag for it. Did you intend to reference the tag instead?

Reply:
I added the download link instead, let me know what do you think.